src/components/Profile/Settings.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { Form } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+// import firebase
+import firebase from '../../firebase/config';
+
+// import action creator for update profile
+import {
+  updateProfileRequest,
+} from '../../store/actions/profileActions';
```

Comment:
Also if you could the aliases provided. ie 'store/actions/profileActions'

Reply:
Done